src/osdc/Objecter.cc

```diff
@@ -3007,13 +3015,15 @@ void Objecter::handle_osd_op_reply(MOSDOpReply *m)
   if (op->oncommit_sync) {
```

Comment:
may as well, although in reality we never set the callback on a non-commit request.